packages/preferences-persistence/src/migrations/preferences-package-data/convert-editor-settings.js

```diff
@@ -17,11 +17,19 @@ export default function convertEditorSettings( data ) {
 			};
 			delete newData[ 'core/edit-post' ][ setting ];
 		}
+
+		if ( data?.[ 'core/edit-post' ]?.[ setting ] !== undefined ) {
```

Comment:
Should it be 'core/edit-site' here?

Reply:
Though potentially the preference for the site editor can be deleted at the same time as the data for the post editor (after line 18) given it's being disregarded and no longer used for anything.

Reply:
oops, right, I'll fix it in one of the other PRs

Reply:
It can't be on line 18 though because it might exist while the post editor one won't.

Reply:
Fix included here https://github.com/WordPress/gutenberg/pull/57480/commits/429a995c38a4e18101226d9498d9bf8e12029cdc